docs/development/compare_atomic_files.ipynb

```diff
@@ -6,145 +6,8878 @@
    "source": [
```

Comment:
<p>I would consider moving this long piece of code outside of the notebook. At least to me, the point of the notebook is to visualise the differences (and to illustrate how to use the comparison code).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>